website/content/terminal/common/_index.md

```diff
@@ -1,10 +1,31 @@
 ---
+title: Common Menus & The User Data Folder
+keywords: "user, files, openbbuserdata, presets, file, portfolio, exports, imports, settings, styles, xlsx, csv, json, themes, openbb, bb, terminal, google, amazon, apple, microsoft, github, opensource, python, markets"
+excerpt: "Common Menus & The User Data Folder in the OpenBB Terminal"
 geekdocCollapseSection: true
 ---
 
-The Common menu is a menu that serves the purpose of being able to make multiple analysis techniques available
-on multiple menus within the Terminal. E.g. technical and quantitative analysis is available for both stocks and
-cryptocurrencies.
+There are several sub-menus which contain features that can be applied across all asset classes. These are grouped in the directory trees under, common.
 
-**Available commands**
-{{< toc-tree >}}
+  - <a href="https://openbb-finance.github.io/OpenBBTerminal/terminal/common/ba/" target="_blank">Introduction to the Behavioural Analysis Menu</a>
+  - <a href="https://openbb-finance.github.io/OpenBBTerminal/terminal/common/qa/" target="_blank">Introduction to the Quantitative Analysis Menu</a>
+  - <a href="https://openbb-finance.github.io/OpenBBTerminal/terminal/common/ta/" target="_blank">Introduction to the Technical Analysis Menu</a>
+  - <a href="https://openbb-finance.github.io/OpenBBTerminal/terminal/forecast/" target="_blank">Introduction to the Forecast Menu</a>
+
+<h1>The OpenBBUserData Folder</h1>
+
+<img width="1117" alt="The settings menu" src="https://user-images.githubusercontent.com/85772166/195736718-a1b821da-5977-437a-bd18-b44add2a29a2.png">
+
+The `OpenBBUserData` folder's default location is the home of the system user account. The location of this folder can be set by the user from the `/settings` menu. This folder is where exported files are saved, it is also where files being imported to the Terminal functions are kept. These include all things user-created:
+ 
+ - Screener presets
+ - Portfolio files
+ - Exported files
+ - Files to be imported by various functions
+ - Styles and themes
+ - Preferred data sources
+ - Anonymized useage logs
+
+<img width="1231" alt="Screen Shot 2022-10-13 at 6 45 01 PM" src="https://user-images.githubusercontent.com/85772166/195742985-19f0e420-d8f7-4fea-a145-a0243b8f2ddc.png">
```

Comment:
Let's swap this image with the one above. I think it is a fair assumption to make that most users don't change the path so this image is more relevant to show first.